libraries/AP_Logger/LogFile.cpp

```diff
@@ -149,7 +149,7 @@ void AP_Logger::Write_RCIN(void)
     };
     WriteBlock(&pkt, sizeof(pkt));
 
-    const uint16_t override_mask = rc().get_override_mask();
```

Comment:
Just a consistency thing. But we use `rc().get_radio_in(values, ARRAY_SIZE(values));` on Line 131?

Reply:
get_radio_in() is not a static function !

Reply:
Of course sorry. Brain malfunction. 

My brain somehow saw you change as, `AP::rc_channels()` instead. LOL  I was like that is weird its the same thing but whatever...